src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -647,7 +654,7 @@ static bool pick_from_internal_rr_locked(
     if (server->drop) {
       // Not using the RR policy, so unref it.
       if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
-        gpr_log(GPR_INFO, "Unreffing RR for drop (0x%" PRIxPTR ")",
+        gpr_log(GPR_INFO, "[grpclb %p] Unreffing RR %p for drop", glb_policy,
                 (intptr_t)wc_arg->rr_policy);
```

Comment:
I don't think we need the `intptr_t` cast here.

Reply:
Done